create-custom-map-style/.nvmrc

```diff
@@ -0,0 +1 @@
+12.21.0
```

Comment:
Is this a specific version requirement for Maputnik? If so, relax it to `12`, otherwise consider removing it.